client/authority-discovery/Cargo.toml

```diff
@@ -21,7 +21,7 @@ codec = { package = "parity-scale-codec", version = "3.0.0", default-features =
 futures = "0.3.21"
 futures-timer = "3.0.1"
 ip_network = "0.4.1"
-libp2p = { version = "0.49.0", default-features = false, features = ["kad"] }
```

Comment:
Why did you enable default features?

Reply:
libp2p doesn't have default features? https://docs.rs/crate/libp2p/0.50.0/features "This version has 62 feature flags, 0 of them enabled by default."